panc-docs/source/pan-book/pan-book.rst

```diff
@@ -2140,6 +2140,18 @@ children can be a full type specification and may contain default values
 and/or validation blocks. One can also attach default values or
 validation blocks to the record as a whole.
 
+The choice-type
+~~~~~~~~~~~~~~~
+The choice type is an advanced type that can be used to create user-defined types.
+It takes two or more strings as input arguments. These arguments represent the
```

Comment:
it's true that choice with one argument makes not much sense, but it should be possible (i assume it is). re rephrase this to `one or more`